server/sonar-web/src/main/js/application.js

```diff
@@ -474,6 +474,31 @@ function closeModalWindow () {
 
 
 /*
+ * Users
+ */
+
+(function() {
+
+  /**
+   * Convert the result of api/users/search to select2 format
+   */
+  window.usersToSelect2 = function (response) {
+    return {
```

Comment:
:new: ![MINOR](https://raw.githubusercontent.com/henryju/image-hosting/master/minor.png)Add a semicolon at the end of this statement.[![...](https://raw.githubusercontent.com/henryju/image-hosting/master/more.png)](http://nemo.sonarqube.org/coding_rules#rule_key=javascript%3ASemicolon)